ufls_detection/example_run_ufls_detection_voltage.R

```diff
@@ -0,0 +1,13 @@
+source("load_tool_environment.R")
+
+db_file = "20210525_Tesla.db"
```

Comment:
Question for Jack / I need to go and read the style guide: when do we want to use = and when do we want to use <- ?
From a very brief google it looks like this may matter...sometimes but not always.

Reply:
- [ ] correct `=` to `<-`